src/app/admin/admin.component.html

```diff
@@ -1,4 +1,5 @@
 <div *ngIf="globalVars.nodeInfo != null">
+  {{ setTitle('Admin - BitClout') }}
```

Comment:
Please move all setTitle calls from html files to `ngOnInit`

Reply:
Will do!